coordinator/pom.xml

```diff
@@ -1,19 +1,56 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
+  <!-- Artifact props -->
   <groupId>io.stargate</groupId>
   <artifactId>stargate</artifactId>
```

Comment:
Could and should we change this to `coordinator-parent`?

Reply:
+1 -- yes, I think that's a good idea.

Reply:
One thing to consider -- C2 extends this parent pom. Still probably a good idea but need to update on that side too.

Reply:
OK but we are in a control there.. We would only break some project that depends on this one, not sure if there is any?

What's a bit shitty is that we did not do this from the beginning of v2. The V1 will continue to have `stargare` here, and V2 would have `coordinator-parent`. Not sure if this really makes sense?

@jeffreyscarpenter wdyt?

Reply:
Maybe we should keep as is. There are other projects that could need to be changed. We would just put extra work on us.

Reply:
Fair point. If we feel strongly about this we can change it at a later point.

The use of this parent pom for C2 itself is bit odd choice anyway; if this was to be changed it would probably make sense as a later refactoring of Stargate/C2 split. So leaving it as-is for now actually makes sense.

Reply:
agree with leaving as is